packages/cli/src/modules/insights/database/repositories/insights-by-period.repository.ts

```diff
@@ -21,6 +21,32 @@ const summaryParser = z
 	})
 	.array();
 
+const aggregatedInsightsByWorkflowParser = z
+	.object({
+		workflowId: z.string(),
+		workflowName: z.string().optional(),
+		projectId: z.string().optional(),
+		projectName: z.string().optional(),
```

Comment:
Under what circumstances are these undefined?

Reply:
You're right, no reason, I'll fix it